etl/steps/data/garden/wb/2023-12-29/gender_statistics.py

```diff
@@ -0,0 +1,219 @@
+    tb = tb.reset_index()
+
+    # Add metadata by finding the descriptions and sources using indicator codes
+    tb = add_metadata(tb, metadata_tb)
+
+    # Set an appropriate index and sort.
+    tb = tb.underscore().set_index(["country", "year"], verify_integrity=True).sort_index().sort_index(axis=1)
+
+    # Combine maternity and paternity leave indicators as described in the issue here: https://github.com/owid/owid-issues/issues/1274
+    tb["total_maternity_leave"] = tb["sh_mmr_leve"] + tb["sh_par_leve_fe"]
+    tb["total_paternity_leave"] = tb["sh_par_leve_ma"] + tb["sh_ptr_leve"]
+
+    # Add relevant metadata to the newly created columns
+    tb["total_maternity_leave"].metadata.description_from_producer = (
+        "**This indicator is a sum of two different maternity leave indicators provided by World Bank:**"
+        + "\n\nThe first indicator is called "
+        + tb["sh_mmr_leve"].metadata.title
+        + ". The revelavant description from World Bank of this indicator is:\n\n"
+        + tb["sh_mmr_leve"].metadata.description_from_producer
+        + "\n\nThe first indicator is called "
+        + tb["sh_par_leve_fe"].metadata.title
+        + ". The revelavant description from World Bank of this indicator is:\n\n"
+        + tb["sh_par_leve_fe"].metadata.description_from_producer
+    )
+
+    tb["total_paternity_leave"].metadata.description_from_producer = (
+        "**This indicator is a sum of two different paternity leave indicators provided by World Bank:**"
+        + "\n\nThe first indicator is called "
+        + tb["sh_par_leve_ma"].metadata.title
+        + ". The revelavant description from World Bank of this indicator is:\n\n"
+        + tb["sh_par_leve_ma"].metadata.description_from_producer
+        + "\n\nThe second indicator is called "
+        + tb["sh_ptr_leve"].metadata.title
+        + ". The revelavant description from World Bank of this indicator is:\n\n"
+        + tb["sh_ptr_leve"].metadata.description_from_producer
+    )
+
+    tb["total_paternity_leave"].metadata.title = "Total days of leave available for the father"
+    tb["total_maternity_leave"].metadata.title = "Total days of leave available for the mother"
+
+    tb[
+        "total_maternity_leave"
+    ].metadata.description_processing = "The total days of leave available for the mother indicator was created by OWID by combining maternity leave and parental leave mother quota indicators. For more details on each indicator see the section on **How producer describes this data**."
+    tb[
+        "total_paternity_leave"
+    ].metadata.description_processing = "The total days of leave available for the father indicator was created by OWID by combining paternity leave and parental leave father quota indicators. For more details on each indicator see **How producer describes this data**."
+
+    for leave_col in ["total_paternity_leave", "total_maternity_leave"]:
```

Comment:
Nit - is there a reason why is this done programatically and not saved in YAML file?

Reply:
good point! I think it's because I wanted to combine `description_from_producer` from the two sources and then just kept going with the other fields.